<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>976</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <widget class="QSplitter" name="middle_line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>921</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QGroupBox" name="group_sort">
     <property name="title">
      <string>Sort</string>
     </property>
     <widget class="QLabel" name="label_sort_col_names">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>31</y>
        <width>72</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Col Name</string>
      </property>
     </widget>
     <widget class="QComboBox" name="combo_sort_col_names">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>30</y>
        <width>281</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="sort_radio_asc">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Asc</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="sort_radio_desc">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>60</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Desc</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sort_sex">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort Sex</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="sort_radio_female">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>90</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="sort_radio_male">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>61</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="sort_radio_all">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>90</y>
        <width>81</width>
        <height>19</height>
       </rect>
      </property>
      <property name="text">
       <string>All</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>130</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_sort_names">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Sort Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_sort_names">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>281</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tbl_view_show_data">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>401</width>
        <height>351</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_info">
     <property name="title">
      <string>Employee Info</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>411</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="layout_grid_info">
       <item row="0" column="0">
        <widget class="QLabel" name="label_info_id">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1" colspan="2">
        <widget class="QPlainTextEdit" name="textedit_memo"/>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_info_memo">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Memo</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QComboBox" name="combo_info_birth_addr"/>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QComboBox" name="combo_info_salary"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QSpinBox" name="spin_info_id"/>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QComboBox" name="combo_info_sex"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_info_name">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Name</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0" rowspan="2">
        <widget class="QLabel" name="label_info_salary">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Salary</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QDateEdit" name="dateedit_brith_year">
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_info_birth_year">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Birth Year</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="lineedit_name"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_info_sex">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Sex</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_info_birth_addr">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Birth Addr</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_info_dept">
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>Dept</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QComboBox" name="combo_info_dept"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>976</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="act_add"/>
   <addaction name="act_insert"/>
   <addaction name="act_del"/>
   <addaction name="separator"/>
   <addaction name="act_new_photo"/>
   <addaction name="act_del_photo"/>
   <addaction name="separator"/>
   <addaction name="act_save"/>
   <addaction name="act_cancel_change"/>
   <addaction name="separator"/>
   <addaction name="act_exit"/>
  </widget>
  <action name="act_add">
   <property name="text">
    <string>Add</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add new line of data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="act_insert">
   <property name="text">
    <string>insert</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;insert new line of data into certain location&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="act_del">
   <property name="text">
    <string>del</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete current line of data&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="act_new_photo">
   <property name="text">
    <string>add_photo</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add new image&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="act_del_photo">
   <property name="text">
    <string>del_photo</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete current photo&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="act_save">
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save changes&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="act_cancel_change">
   <property name="text">
    <string>cancel_change</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Cancel changes&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="text">
    <string>exit</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Quit program&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
